.github/workflows/test.yml

```diff
@@ -23,62 +23,64 @@ jobs:
+        run: docker volume rm bodhijs-postgres || true && docker rm -vf $(docker ps -a | grep bodhijs-subquery-node | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs-graphql-engine | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs-postgres | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs-mandala-node | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs-eth-rpc-adapter-server | awk '{print $1}') || true  && docker rm -vf $(docker ps -a | grep bodhijs-eth-rpc-adapter-server-with-subql | awk '{print $1}') || true  && docker rm -vf $(docker ps -a | grep bodhijs-feed-tx | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs-eth-rpc-adapter-test | awk '{print $1}') || true
       - name: check docker info after cleaning
         run: docker ps && docker volume ls
       - name: build bodhi base # if use self-hosted machine can utilize cache
         run: docker build . -t bodhi-base -f docker/bodhi-base.Dockerfile
       - name: test
-        run: docker-compose up --abort-on-container-exit --exit-code-from=eth-rpc-adapter-test --build -- eth-rpc-adapter-test
+        run: docker compose up --abort-on-container-exit --exit-code-from=eth-rpc-adapter-test --build -- eth-rpc-adapter-test
       - name: clean up after tests finished
-        run: docker volume rm bodhijs_postgres || true && docker rm -vf $(docker ps -a | grep bodhijs_subquery-node | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs_graphql-engine | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs_postgres | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs_mandala-node | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs_eth-rpc-adapter-server | awk '{print $1}') || true  && docker rm -vf $(docker ps -a | grep bodhijs_eth-rpc-adapter-server-with-subql | awk '{print $1}') || true  && docker rm -vf $(docker ps -a | grep bodhijs_feed-tx | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs_eth-rpc-adapter-test | awk '{print $1}') || true
+        run: docker volume rm bodhijs-postgres || true && docker rm -vf $(docker ps -a | grep bodhijs-subquery-node | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs-graphql-engine | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs-postgres | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs-mandala-node | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs-eth-rpc-adapter-server | awk '{print $1}') || true  && docker rm -vf $(docker ps -a | grep bodhijs-eth-rpc-adapter-server-with-subql | awk '{print $1}') || true  && docker rm -vf $(docker ps -a | grep bodhijs-feed-tx | awk '{print $1}') || true && docker rm -vf $(docker ps -a | grep bodhijs-eth-rpc-adapter-test | awk '{print $1}') || true
```

Comment:
`docker compose down -v` should clean everything

Reply:
it removes everything, is it an issue? I remember a while ago we wanted to make sure it doesn't accidentally clear other processes

Reply:
oh so it only clears containers and volumes related to the compose file, so we should be good